src/main/java/baekjoon/barkingdog/queue/Silver2164.kt

```
@@ -0,0 +1,12 @@
+package baekjoon.barkingdog.queue
+
+fun main(){
+    val number = readln().toInt()
+    val queue = ArrayDeque<Int>(number)
+    (1..number).forEach{ queue.add(it) }
```

detekt (1.23.8)

[warning] Using the forEach method on ranges has a heavy performance cost. Prefer using simple for loops.

(detekt.performance.ForEachOnRange)